package.json

```diff
@@ -78,6 +78,7 @@
     "sinon": "^1.15.4",
     "sinon-chai": "^2.8.0",
     "transfer-webpack-plugin": "^0.1.4",
+    "warning": "^2.1.0",
```

Comment:
About this, all other dependencies are also placed in `devDependencies`. Like wouldn't it be inconsistent if I start using `dependencies`?